Improve wrapping of repo header buttons on smaller screens

cheap-glitch opened the pull request:
Fixes #5398 

## Test URLs

[refined-github/refined-github](https://github.com/refined-github/refined-github)
[fregante/webext-dynamic-content-scripts](https://github.com/fregante/webext-dynamic-content-scripts)

## Screenshot

**Before** :-1:

![before](https://user-images.githubusercontent.com/46634000/167182088-82085f2f-b732-4a2e-bac2-70c3c61a95d0.png)

**After** :+1:

![after](https://user-images.githubusercontent.com/46634000/167182078-6c5d332d-d76a-425b-9a3e-b2f506ac936b.png)

cheap-glitch reviewed (commented):
Note: as shown in the video above, this PR also restores the buttons on mobile — where they're normally hidden by GitHub.

Not sure if this is a welcomed change, so I don't mind dropping it.

cheap-glitch commented on source/refined-github.css:
```diff
@@ -177,3 +177,22 @@ pr-branches
+		gap: 8px;
+	}
+
+	.pagehead-actions li {
+		margin-right: 0 !important;
+	}
```
Needed because this is replaced by `gap` on the parent.

fregante replied:
I'm not sure why but I'm seeing this on Chrome

<img width="488" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/167236300-b6209f61-4567-4b03-a3dd-f0fd46ee3cdb.png">

cheap-glitch replied:
That's likely because the `display: flex` didn't apply, the selector just needs to be made more specific.

cheap-glitch reviewed (commented):
I think I've reduced the CSS to the minimum amount necessary. Unfortunately it still requires a bit more than 4 lines.

fregante commented on source/refined-github.css:
```diff
@@ -177,3 +177,22 @@ pr-branches
+
+/* Improve wrapping of repo header buttons on smaller screens #5398 */
+@media (min-width: 768px) {
+	#repository-container-header > :first-child > :first-child {
+		flex-shrink: 0 !important;
+	}
```
We also have to consider long repository names. `flex-shrink: 3` or higher gives priority to the title without causing the opposite problem.

> not shrink at all

That's the exact reason why this bug exist. This PR causes the opposite problem:

<img width="1372" alt="Screen Shot 2" src="https://user-images.githubusercontent.com/1402241/167236225-05c08518-ea5b-4ff4-96e4-3b7ca1646629.png">

We have to allow wrapping, this is too much code to achieve that

cheap-glitch commented:
Changed to the "shrink more strategy" and moved the fix to `github-bugs.css`. Tested on Chromium too:

![image](https://user-images.githubusercontent.com/46634000/167256187-ae9fea99-7740-4071-a4ce-0257e555e815.png)

cheap-glitch commented:
Closing in favor of #5622
